### PR TITLE
Release Apache for Kibana 8

### PR DESCRIPTION
**Pull request by jsoriano**
## What does this PR do?

Release a version of the Apache package compatible with the 8 version of the stack.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/master/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] If I'm introducing a new feature, I have modified the Kibana version constraint in my package's `manifest.yml` file to point to the latest Elastic stack release (e.g. `^7.13.0`).

## Related issues

- https://github.com/elastic/elastic-package/pull/571

### CODE REVIEW

File: packages/apache/manifest.yml [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+version: 1.3.0
 license: basic
 description: Collect logs and metrics from Apache servers with Elastic Agent.
 type: integration
 categories:
   - web
 release: ga
 conditions:
-  kibana.version: "^7.14.0"
+  kibana.version: "^7.14.0 || ^8.0.0"
```
Comment (mtojek) [inline packages/apache/manifest.yml:12]: Let's keep the discussion public.

We know about two options (`^8.0.0` vs `^7.16.0 || ^8.0.0`). I think we should prepare a document/table with both solutions and document pros and cons there, describe what works and what doesn't. Integrations developer can decide which approach they would like to adopt.

Comment (ruflin) [reply packages/apache/manifest.yml:12]: I opted out of `^8.0.0` for now in my PR to give us some time figure out the long term compatibility. But in theory I see that this is the correct option.

Comment (jsoriano) [reply packages/apache/manifest.yml:12]: Started to draft some guidelines for package developers: https://github.com/elastic/elastic-package/pull/571

Comment (jsoriano) [reply packages/apache/manifest.yml:12]: So we are good with this constraint, right? (if the package owners agree :slightly_smiling_face:)

Comment (ruflin) [reply packages/apache/manifest.yml:12]: On my end, 👍 on the change following the assumption package-registry will offer some magic later on ;-)

File: packages/apache/changelog.yml [hunk 2]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.3.0"
+  changes:
+    - description: Support Kibana 8.0
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/
```
Comment (mtojek) [inline packages/apache/changelog.yml:6]: PR reference is missing here.

Comment (jsoriano) [reply packages/apache/changelog.yml]: Good catch.

**Review by mtojek — APPROVED** [review]

**Review by ruflin — APPROVED** [review]

**Pull request merged**